sdk/storage/storage-queue/test/messageidclient.spec.ts

```diff
@@ -90,6 +91,41 @@ describe("MessageIdClient", () => {
     assert.deepStrictEqual(pResult2.peekedMessageItems[0].messageText, newMessage);
   });
 
+  it("update and delete message with all parameters - test sas connection string MessageIdClient constructor", async () => {
+    let messagesClient = queueClient.getMessagesClient();
+    let eResult = await messagesClient.enqueue(messageContent);
+    assert.ok(eResult.date);
+    assert.ok(eResult.expirationTime);
+    assert.ok(eResult.insertionTime);
+    assert.ok(eResult.messageId);
+    assert.ok(eResult.popReceipt);
+    assert.ok(eResult.requestId);
+    assert.ok(eResult.timeNextVisible);
+    assert.ok(eResult.version);
+
+    let newMessage = "New Message";
+    let messageIdClient = new MessageIdClient(
+      getSASConnectionStringFromEnvironment(),
```

Comment:
>getSASConnectionStringFromEnvironment [](start = 6, length = 37)

Can we generate a SAS connection string dynamically using storage account & key in Node.js runtime?

Account Name + Account Key => SAS

So we can reuse current account_name and account_key environment variable, instead of importing a new environment variable.

SAS aims for a short term period access, and can be oudated when storage key is rotated. Also a SAS has a valid period, after that, SAS will be outdated. 

In browser runtime, we can use the environment variable account_name with account_sas to generate the SAS connection string too.

Reply:
Sure, we can do that. I'll update.